src/app/feed/shared/connection-config.model.spec.ts

```diff
@@ -0,0 +1,151 @@
+import {
+  beforeEachProviders,
+  beforeEach,
+  describe,
+  expect,
+  it
+} from "@angular/core/testing";
+
+import { ConnectionConfig, IWanted } from "./connection-config.model";
+
+declare const jasmine;
+declare const spyOn;
+
+describe("Service: ConnectionConfig", () => {
+
+  beforeEachProviders(() => {
+    return [
+      {provide: ConnectionConfig, useClass: ConnectionConfig}
+    ];
+  });
+
+  beforeEach(() => {
+    this.pluginHandle = {
+      send: jasmine.createSpy(": voidpluginHandle.send")
+    };
+  });
+
+  it("should call pluginHandle.send on create", () => {
+    let config: IWanted = {
+      audio: true,
+      video: true,
+    };
+    let cconfig = new ConnectionConfig(
+      this.pluginHandle,
+      config,
+      {},
+      undefined
+    );
+
+    expect(cconfig).toBeDefined();
+    expect(this.pluginHandle.send).toHaveBeenCalled();
+  });
+
+  it("should not be defined until confirm", () => {
```

Comment:
When you write unit tests, you're testing/describing the public API. So I would use `describe` to make clear what you're testing. For example, here you're testing the `get` method.

```javascript
describe("#get", () => {
  it("...")
});
```

On the other hand, with the current test, it's not clear what's expected from the `get` method.